### DISCUSSION TITLE
#520 removes the Azure Devops pipelines is support being removed?

### DISCUSSION THREAD

**Discussion by GarethR-UK** [Azure/Enterprise-Scale#537] [Q&A]
#522 removes the Azure Devops pipelines. Is support being removed or is the intention for this to be achieved in a different way? This instructions for using Azure Devops seems to suggest the intention is that Azure DevOps can be used instead of GitHub/GitHub Enterprise.

**Comment by krnese** [comment]
We are in the midst of cleaning up some docs and references. For AzOps (also for Azure Devops pipelines), see the documentation here: https://github.com/Azure/AzOps-Accelerator

We have also made an integration between Enterprise-Scale deployment and AzOps, but here we only support GitHub as of now (adding support for Azure DevOps is coming): https://github.com/Azure/Enterprise-Scale/wiki/Deploying-Enterprise-Scale#platform-devops-and-automation
